tests/webapp/api/test_bug_job_map_api.py

```diff
@@ -148,3 +148,17 @@ def test_bug_job_map_delete(webapp, eleven_jobs_stored, test_repository,
         content = json.loads(resp.content)
         assert content == {"message": "Bug job map deleted"}
         assert BugJobMap.objects.count() == 0
+
+
+def test_bug_job_bad_job_id(webapp, test_repository):
```

Comment:
Perhaps name the test `test_bug_job_map_bad_job_id`?

Other than that looks great - I'll merge once the test name is updated :-)